bitset.go

```diff
@@ -192,6 +192,63 @@ func (b *BitSet) Flip(i uint) *BitSet {
 	return b
 }
 
+// Shrink shrinks BitSet to desired length
```

Comment:
Maybe "'shrinks BitSet to desired length "in bits" (add: in bits)

(I just want your opinion on this proposal, this is not a change request.)

Reply:
`in bits` seems like a good idea. I actually thought about this before creating the PR. The reason I left it out is because it feels a little bit misleading since there will more than likely be an extra uint64 on the end of it. I'm fine with either way.